netbox_netprod_importer/push.py

```diff
@@ -141,6 +144,20 @@ def _push_interfaces(self):
                 self._handle_interface_mode(interface, if_prop["mode"])
                 if_prop.pop("mode")
 
+            if if_prop["untagged_vlan"]:
+                vlan_id = self._get_vlan_id(if_prop["untagged_vlan"])
+                if vlan_id != -1:
+                    setattr(interface, "untagged_vlan", vlan_id)
+
+            if_prop.pop("untagged_vlan")
```

Comment:
You'll get a `KeyError` if `untagged_vlan` is not defined.

Reply:
"untagged_vlan" is always there, is being formed: importer.py:176